scalene/__main__.py

```diff
@@ -1,12 +1,14 @@
 import sys
 
+
 def main():
     try:
-        from scalene import scalene
-        scalene.Scalene.main()
+        from scalene import entrypoint
```

Comment:
Maybe would be better to call it `profiler`? (`entrypoint` is pretty nondescript.)

Reply:
Or even `scalene_profiler`...